plugins/ui/make_icon_types.py

```diff
@@ -0,0 +1,80 @@
+import re
+
+
+def camel_to_snake(name: str) -> str:
+    s1 = re.sub("(.)([A-Z][a-z]+)", r"\1_\2", name)
+    return re.sub("([a-z0-9])([A-Z])", r"\1_\2", s1).lower()
+
+
+relative_path = "./src/js/node_modules/@deephaven/icons/dist/index.d.ts"
+
+icons = {}
+snakeCase = {}
+noPrefix = {}
+snakeCaseNoPrefix = {}
+
+with open(relative_path, "r") as file:
+    lines = file.readlines()
+    for line in lines:
```

Comment:
Instead of reading the entire file at once, you can read line by line:
```suggestion
    for line in file:
```
It likely doesn't matter too much in this case since the input file is not that big, but if this were a large file reading it all into memory could cause issues. Best to just read it line by line.